### PR TITLE
Add helper to get the xtask command

### PR DESCRIPTION
**Pull request by cecton**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -156,6 +157,11 @@ pub fn package(name: &str) -> Option<&cargo_metadata::Package> {
     metadata().packages.iter().find(|x| x.name == name)
 }
 
+/// Return a [`std::process::Command`] to the xtask command currently running.
```
Comment (cecton) [inline src/lib.rs:160]:
```suggestion
/// Return a [`std::process::Command`] of the xtask command currently running.
```

Comment (yozhgoor) [inline src/lib.rs]:
```suggestion
/// Return a [`std::process::Command`] of the xtask command currently running.
```

Comment (yozhgoor) [reply src/lib.rs]: Lucky LuQ

Comment (cecton) [reply src/lib.rs]: :rofl:

**Review by yozhgoor — APPROVED** [review]

**Pull request merged**